ethcore/src/ethereum/ethash.rs

```diff
@@ -453,11 +453,14 @@ impl Ethash {
 		let parent_has_uncles = parent.uncles_hash() != &sha3::SHA3_EMPTY_LIST_RLP;
 
 		let min_difficulty = self.ethash_params.minimum_difficulty;
+
 		let difficulty_hardfork = header.number() >= self.ethash_params.difficulty_hardfork_transition;
-		let difficulty_bound_divisor = match difficulty_hardfork {
-			true => self.ethash_params.difficulty_hardfork_bound_divisor,
-			false => self.ethash_params.difficulty_bound_divisor,
+		let difficulty_bound_divisor = if difficulty_hardfork {
+			self.ethash_params.difficulty_hardfork_bound_divisor
+		} else {
```

Comment:
A quick grep through the codebase shows only 2 or 3 uses of `= match ... { true => ..., false => ... }` and over a hundred uses of `= if` (I might have missed a couple because I have to use `-A 1` and eyeball it for the `match` case)